eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/ImpTest.hs

```diff
@@ -222,16 +236,21 @@ instance
   ) =>
   ShelleyEraImp (ConwayEra c)
   where
-  initImpNES =
+  initImpNES = do
     let nes =
           initAlonzoImpNES
             & nesEsL . curPParamsEpochStateL . ppDRepActivityL .~ EpochInterval 100
             & nesEsL . curPParamsEpochStateL . ppGovActionLifetimeL .~ EpochInterval 30
+            & nesEsL . curPParamsEpochStateL . ppCommitteeMaxTermLengthL .~ EpochInterval 20
+            & nesEsL . epochStateGovStateL . committeeGovStateL .~ SJust initialCommittee
+        -- TODO: configure sensible voting thresholds, rather than defaulting to zero
+        -- in order to make voting in tests relevant
         epochState = nes ^. nesEsL
         ratifyState =
           def
             & rsEnactStateL .~ mkEnactState (epochState ^. epochStateGovStateL)
-     in nes & nesEsL .~ setCompleteDRepPulsingState def ratifyState epochState
+    void $ addKeyPair initialCommitteeKeyPair
```

Comment:
Let's generate it at random. You'll just need to add `MonadGen`  constraint to `initImpNES`
```suggestion
    committeeKeyPair <- freshKeyPair
```